### PR TITLE
Goal sequence driver - automated

### PR DESCRIPTION
**Pull request by padhupradheep**

### CODE REVIEW

File: src/automated_sequence.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+				self.data.append([{'X': self.goal_list[i][0].pose.position.x, \
+					'Y': self.goal_list[i][0].pose.position.y,\
+					'theta': self.goal_list[i][0].pose.orientation.z}, 
+			])
+				
+				
+			with open(goal_list, 'w') as outfile:
+				for i in range(0, len(self.goal_list)):
+					yaml.dump(self.data[i], outfile, default_flow_style=False)
+				# yaml.dump(self.data[1], outfile, default_flow_style=False)
```
Comment (madMAx43v3r) [inline src/automated_sequence.py]: can we remove that line?

**Review by madMAx43v3r — CHANGES_REQUESTED** [review]

File: src/automated_sequence.py (hunk 1 above)
Comment (padhupradheep) [reply src/automated_sequence.py]: Yes we could do that

Comment (madMAx43v3r) [reply src/automated_sequence.py]: Yeah, it seems that was for debugging only. Thx.

Comment (padhupradheep) [reply src/automated_sequence.py]: Amended the change with some more extra comments! I think we can merge the pull request !

**Pull request merged**